states/base.py

```diff
@@ -16,6 +16,7 @@ def __init__(self):
         super().__init__()
         self.killed = False
         self.on_pi = which("rpi-update")
+        self.is_game = False
```

Comment:
is_game kan dan weg -> het zetten van player_class betekent dus dat het een game is